pom.xml

```diff
@@ -254,9 +258,9 @@
             <plugin>
                 <groupId>io.fabric8</groupId>
                 <artifactId>docker-maven-plugin</artifactId>
-                <version>0.39.0</version>
+                <!-- Please do not upgrade to 0.40.2 as there is an issue with multi-platforms support -->
```

Comment:
Can you link it, so that we know when it is safe to upgrade?

Reply:
We can upgrade to `0.40.2`. I see that all docker images (for two platforms) are pushed correctly.